### PR TITLE
Block pool list page showing empty option on kebab action menu

### PR DESCRIPTION
**Pull request by GowthamShanmugam**
BZ Ref: https://bugzilla.redhat.com/show_bug.cgi?id=2099646

Signed-off-by: Gowtham Shanmugasundaram <gshanmug@redhat.com>

### CODE REVIEW

**Comment by GowthamShanmugam** [comment]
image

File: packages/odf/components/mcg/ObjectBucketClaim.tsx [hunk 1]
```diff
@@ -219,7 +219,9 @@ const OBCRow: React.FC<RowProps<K8sResourceKind, CustomData>> = ({
             namespace,
           }}
           customKebabItems={(t) => ({
-            ATTACH_DEPLOYMENT: t('Attach to Deployment'),
+            ATTACH_DEPLOYMENT: {
+              value: t('Attach to Deployment'),
+            },
```
Comment (bipuladh) [inline packages/odf/components/mcg/ObjectBucketClaim.tsx:224]:
```suggestion
```
Already done by https://github.com/red-hat-storage/odf-console/pull/255

**Review by bipuladh — CHANGES_REQUESTED** [review]

**Comment by GowthamShanmugam** [comment]
Screenshot from 2022-06-21 12-16-18

File: packages/ocs/block-pool/BlockPoolDetailsPage.tsx [hunk 2]
```diff
@@ -49,7 +50,7 @@ export const BlockPoolDetailsPage: React.FC<BlockPoolDetailsPageProps> = ({
     },
     {
       name: t('StorageSystem details'),
-      path: location.pathname.split(`/${kind}`)[0],
+      path: `${location.pathname.split(`/${kind}`)[0]}/overview/block-file`,
```
Comment (bipuladh) [inline packages/ocs/block-pool/BlockPoolDetailsPage.tsx:53]:
```suggestion
      path: `${location.pathname.split(`/${kind}`)[0]}/${kind}`,
```

**Review by bipuladh — CHANGES_REQUESTED** [review]

File: packages/ocs/block-pool/BlockPoolDetailsPage.tsx (hunk 2 above)
Comment (GowthamShanmugam) [reply packages/ocs/block-pool/BlockPoolDetailsPage.tsx:53]: ack

**Pull request merged**